proxmoxtf/provider/provider.go

```diff
@@ -129,12 +129,17 @@ func providerConfigure(_ context.Context, d *schema.ResourceData) (interface{},
 		sshConf[mkProviderSSHAgentSocket] = sshAgentSocket
 	}
 
-	nodeOverrides := map[string]string{}
+	nodeOverrides := map[string]ssh.ProxmoxNode{}
 
 	if ns, ok := sshConf[mkProviderSSHNode]; ok {
 		for _, n := range ns.([]interface{}) {
 			node := n.(map[string]interface{})
-			nodeOverrides[node[mkProviderSSHNodeName].(string)] = node[mkProviderSSHNodeAddress].(string)
+			ProxmoxNode := ssh.ProxmoxNode{
+				Address: node[mkProviderSSHNodeAddress].(string),
+				Port:    int32(node[mkProviderSSHNodePort].(int)),
```

Comment:
really minor, to simply a bit:
```suggestion
				Port:    node[mkProviderSSHNodePort].(int32),
```